clients/ctl/admin-ui/src/features/common/BorderGrid.tsx

```diff
@@ -0,0 +1,45 @@
+import { chunk } from "@chakra-ui/utils";
+import { Box, SimpleGrid } from "@fidesui/react";
+import { ReactNode } from "react";
+
+import classes from "./BorderGrid.module.css";
+
+interface Props<T> {
+  columns: number;
+  items: T[];
+  renderItem: (item: T) => ReactNode;
+}
+const BorderGrid = <T extends unknown>({
+  columns,
+  items,
+  renderItem,
+}: Props<T>) => {
+  const chunks = chunk(items, columns);
+
+  return (
+    <Box>
+      {chunks.map((parent, index, { length }) => (
+        <Box
+          key={JSON.stringify(parent)}
+          className={classes["grid-row"]}
```

Comment:
I know this is all pretty new and we don't have set standards quite yet, but I'm just noting/wondering about using classes in this way versus using css in jsx/tsx

Reply:
yeah, I had the same thought. I copied this from [fidesops](https://github.com/ethyca/fidesops/commit/2f38ec19985b7732d7e1092bf2c52b9d008be950) and thought it'd be good to keep it the same since we are merging the UI's soon. I'm guessing this css is a little more complicated than our typical css which is why it was pulled into its own file (as far as I can tell, it's the only css file in fidesops). maybe @chriscalhoun1974 can lend some insight?

Reply:
Hey! My approach was to make the CSS locally scoped within the ConnectionGrid component which would prevent naming clashes. The CSS classes are encapsulated within the ConnectionGrid component. Don't want any 3rd party css classes to override these css classes. Obviously if the css classes were to be reused, I would have created a global css file. Let me know if you have any questions.

Here is a article for reference: https://medium.com/@ralph1786/using-css-modules-in-react-app-c2079eadbb87

Reply:
Thanks Chris! I guess the question was more about whether the CSS file could have just been included in the tsx file via chakra's style props. My guess was that the `nth-child` stuff made it more complicated so it was easier to just pull it into a CSS file? I guess it could also have been passed in via a `styles` or `css` prop as an alternative?

Reply:
Yes, the nth-child selector was easier to implement and more legible to read.